src/background.ts

```diff
@@ -73,12 +75,17 @@ const browsingDataCleanup = async (
 const onSettingsChange = async () => {
```

Comment:
Maybe refactoring the onSettingsChange stuff to a separate file just to shorten `background.ts` a bit?

Reply:
Extraction attempted via a5cfdb7